crates/aptos/e2e/cases/stake.py

```diff
@@ -356,3 +356,63 @@ def test_stake_withdraw_stake_after_unlock(run_helper: RunHelper, test_name=None
         raise TestError(
             f"The stake should be decreased by {amount_to_withdraw}. Expected {current_stake - amount_to_withdraw}, got {result[0].get('total_stake')}"
         )
+
+
+@test_case
+def test_stake_request_commisson(run_helper: RunHelper, test_name=None):
+    # create a new account
+    run_helper.run_command(
+        test_name,
+        [
+            "aptos",
+            "init",
+            "--profile",
+            "request_comisson",
```

Comment:
```suggestion
            "request_comission",
```

Reply:
I won't do the rest of them, there's about 6 more haha

Reply:
oops good catch!